### PR TITLE
Add loged-in check

### PR DESCRIPTION
**Pull request by aintgoin2goa**

### CODE REVIEW

File: test/user.spec.js [hunk 1]
```diff
@@ -240,5 +240,28 @@ describe('Next User Model', function() {
...
+			});
+
+			it('should return false when FT_P not present in cookie', function() {
+				var user = new User({ cookiesStr: userFixtures.cookies[3] });
+				expect(user.isLoggedIn()).to.equal(false);
+			});
+
+			it('should return true when FT-P header is present', function() {
+				var user = new User({ headersObj: userFixtures.headers[3] });
+				expect(user.isLoggedIn()).to.equal(false);
```
Comment (keirog) [inline test/user.spec.js]: true/false mismatch?

File: test/user.spec.js [hunk 2]
```diff
@@ -240,5 +240,28 @@ describe('Next User Model', function() {
...
+			});
+
+			it('should return true when FT_P is present in cookie', function() {
```
Comment (keirog) [inline test/user.spec.js]: Same

**Comment by keirog** [comment]
:+1: apart from above mismatches

**Comment by aintgoin2goa** [comment]
thanks, have fixed the tests

**Pull request merged**